s5a/__init__.py

```diff
@@ -97,6 +97,20 @@ def point_to_h3(dataframe, resolution=1):
     return dataframe
 
 
+def h3_to_point(dataframe, h3_column='h3'):
+    """Convert h3 index in dataframe into longitude and latitude.
+
+    :param: dataframe: pandas dataframe with a column of h3 indices
+    :type dataframe: pandas.core.frame.DataFrame
+    :param h3_column: column name of the column with the h3 indices,
+                      defaults to h3.
+    :type h3_column: str, optional
+    """
+    dataframe[['longitude', 'latitude']] = [
+        h3.h3_to_geo(h3grid) for h3grid in dataframe['h3']]
```

Comment:
It's not a grid but a hexagon, isn't it?

```suggestion
        h3.h3_to_geo(h3hexagon) for h3hexagon in dataframe['h3']]
```